### DISCUSSION TITLE
Java 14 antler grammar

### DISCUSSION THREAD

**Discussion by quantrpeter** [apache/netbeans#5699] [General]
Seem no one build Java 14 antler grammar, why Netbeans can provide syntax highlighting and error highlight?
thanks

**Comment by AdrianVuncannonD** [comment]
NetBeans likely provides syntax highlighting and error highlighting for Java 14 code using a combination of approaches.

**Comment by armstrongwalkley** [comment]
Netbeans, Antlr gramerleri için genel desteği ve yerleşik Java 14 sözdizimi desteği aracılığıyla Java 14 Antlr dilbilgisi için sözdizimi vurgulama ve hata vurgulama sağlar. Bu, hiç kimse Netbeans için özel olarak bir Java 14 Antlr dilbilgisi oluşturmamış olsa da, yazılımın genel yeteneklerine dayalı olarak buna destek sağlayabildiği anlamına gelir.
